src/rgw/driver/sfs/sqlite/dbconn.h

```diff
@@ -202,24 +204,64 @@ inline auto _make_storage(const std::string& path) {
 
 using Storage = decltype(_make_storage(""));
 
+static void sqlite_error_callback(void* ctx, int error_code, const char* msg) {
+  const auto cct = static_cast<CephContext*>(ctx);
+  lderr(cct) << "[SQLITE] (" << error_code << ") " << msg << dendl;
+}
+
+static int sqlite_profile_callback(
```

Comment:
could be useful printing also the thread-id?

Reply:
RGW already does this:

```
2023-07-24T18:12:42.100+0200 *7f21d6d5cf80*  0 framework: beast
```
The hex string after the timestamp is the `pthread_t`